src/main/java/de/muenchen/isi/api/dto/calculation/PlanungsursachlicheWohneinheitenDto.java

```diff
@@ -11,7 +11,7 @@ public class PlanungsursachlicheWohneinheitenDto {
     private String foerderart;
 
     @NotNull
-    private Integer jahr; // JJJJ
+    private String jahr;
```

Comment:
Weshalb wurde aus dem Integer ein String?

Reply:
In der Serviceklasse habe ich es jetzt gesehen. 

Der Grund ist so wie ich das aus dem Code lese, dass auch die 10-, 15- und 20-Jahressummen bzw. Mittelwerte damit abgebildet werden können.